### PR TITLE
feat: spotless integration proposal

### PR DESCRIPTION
**Pull request by chemarubio**

### CODE REVIEW

File: docs/DEV_README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ See [**Frontitude CLI**](./FRONTITUDE_CLI.md) to know how to update your `string
 - [__Remote reference__](./REMOTE_PUBLISHING.md)
 - [__Workflow reference__](./ACTION_PUBLISHING.md)
 
+## Spotless integration
+
+[Spotless](https://github.com/diffplug/spotless) plugin for Gradle.
+
+Run ./gradlew spotlessCheck to detect issues or ./gradlew spotlessApply to apply fixing for the issues detected automatically.
```
Comment (miguelbcr) [inline docs/DEV_README.md:40]: you can use this quotes \`  \`  wrapping the text you want to display as command, like `./gradlew spotlessCheck` and  `./gradlew spotlessApply`, so it is easier to see the commands

**Review by miguelbcr — APPROVED** [review]
nice! thanks! 👏 we can tweak the rules if needed later

File: docs/DEV_README.md (hunk 1 above)
Comment (chemarubio) [reply docs/DEV_README.md]: True ! Forgot to highlight the commands ! Thanks !

**Comment by chemarubio** [comment]
Will merge and then think about possible improvements we came up together during our meeting 👍

**Pull request merged**